Gemfile.lock

```diff
@@ -642,4 +642,4 @@ DEPENDENCIES
   whenever
 
 BUNDLED WITH
-   1.17.3
+   2.4.5
```

Comment:
I think, since there are now no changes to the `Gemfile` left, `Gemfile.lock` should remain untouched too. right?

Reply:
yess you're right!